lib/bootloader_setup.pm

```diff
@@ -73,7 +73,11 @@ use constant GRUB_DEFAULT_FILE => "/etc/default/grub";
 # prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
 # 'up' also works in textmode and UEFI menues.
 sub stop_grub_timeout {
-    send_key 'up';
```

Comment:
just in case? this is covered by send_key_until_needlematch. original it just uses send_key 'up' without knowing if it is on grub2, bootloader menu... it is not okay to me.